node/test/polkadot-simnet/common/src/lib.rs

```diff
@@ -344,11 +354,14 @@ where
 		.events()
 		.into_iter()
 		.filter(|event| match event.event {
-			Event::Democracy(democracy::Event::Passed(_index)) if _index == ref_index => true,
-			Event::Democracy(democracy::Event::PreimageUsed(_hash, _, _))
+			Event::Democracy(democracy::Event::Passed { ref_index: _index })
+				if _index == ref_index =>
+				true,
```

Comment:
This should've just used:

```suggestion
			Event::Democracy(democracy::Event::Passed { ref_index: _index }) => _index == ref_index,
```

instead of using a match guard. Same for the ones below.

Reply:
yes, but better not to try to refactor too much in these kind of PR, in case one refactor is wrong, it will be easily missed.
I like that this PR only does what it is supposed to.